frontend/src/components/LowerButtonNavBar.js

```diff
@@ -36,20 +23,11 @@ export default function LowerButtonNavBar() {
                         setValue(newValue);
                     }}
                 >
-                    <BottomNavigationAction label="Home" icon={<RestoreIcon />} />
-                    <BottomNavigationAction label="Spot" icon={<FavoriteIcon />} />
-                    <BottomNavigationAction label="Wallet" icon={<ArchiveIcon />} />
+                    <BottomNavigationAction label="Home" icon={<RestoreIcon href="/" />} />
+                    <BottomNavigationAction label="Spot Trading" icon={<TrendingUpIcon />} />
+                    <BottomNavigationAction label="Wallet" icon={<AccountBalanceWalletIcon href="/wallet" />} />
```

Comment:
you could as well hav eBottomNavigationAction determine which icon to choose depending on the label instead of passing the icon components around

Reply:
meanwhile changed, thanks